tests/annotations/tests.py

```diff
@@ -120,17 +120,62 @@ def test_basic_annotation(self):
         for book in books:
             self.assertEqual(book.is_book, 1)
 
+    def test_basic_alias(self):
+        books = Book.objects.alias(
+            is_book=Value(1, output_field=IntegerField())
```

Comment:
No need for `Value` wrapping since 1e38f1191de21b6e96736f58df57dfb851a28c1f

```suggestion
            is_book=1,
```

Ditto for all `Values` uses below.

Reply:
Without `Value` there is an error `TypeError: QuerySet.annotate() received non-expression(s): 1.`. Am I missing something?

Reply:
Ahh looks like you'll need to keep passing `Value` in this case but you can drop the `output_field`.
```suggestion
            is_book=Value(1)
```